OpenNMT/train.py

```diff
@@ -95,11 +96,15 @@
 #                     help="Seed for random initialization")
 
 opt = parser.parse_args()
+opt.cuda = len(opt.gpus)
```

Comment:
Can you remove this? You can use `opt.gpus` directly for bool comparisons (since [] evaluates to false), and use `len(opt.gpus)` directly elsewhere.